xbmc/pvr/PVRPlaybackState.cpp

```diff
@@ -359,6 +359,9 @@ void CPVRPlaybackState::StartPlayback(CFileItem* item,
 
     if (item->IsPVRChannel())
     {
+      if (source == PVR_SOURCE::DEFAULT)
+        client->StreamClosed();
```

Comment:
add error handling

Reply:
Added a log statement in the event the clint returns an error. Is that sufficient?